README.md

```diff
@@ -19,14 +19,53 @@ import React from 'react';
 import LazyRender from 'react-lazily-render';
 
 <div>
-  lots of content...
+  ...lots of content...
+  <LazilyRender
+    placeholder={<PlaceholderComponent/>}
+    content={<ExpensiveComponent/>}
+  />
+  ...lots of content...
   <LazilyRender>
     {render => render
       ? <ExpensiveComponent/>
       : <PlaceholderComponent/>
     }
   </LazilyRender>
-  lots of content...
+  ...lots of content...
 </div>
 
 ```
+
+## API
+
+### Properties
+
+#### className
+
+> `string`
+
+The `className` applied to the wrapping element.
+
+#### placeholder
+
+> `React.Node`
+
+Rendered when the component hasn't been scrolled into view.
+
+#### content
```

Comment:
Only due to the fact that `children` is already used for function as a child. So using that same logic `placeholder` isn't necessary either because you can always use the render function.

I added these props because they make it easier for the consumer test. I left the function as a child in there and added the `placeholder` and `content` props similar to [`react-render-image`](https://github.com/jameslnewell/react-render-image) which implements two modes - simple where you can use the props (and is easier to test) - or advanced by using the render function and extra data you get... but since there's not really any extra data provided in this component's render function, do you think I should just remove it?